docs/sql-ref-null-semantics.md

```diff
@@ -236,7 +236,7 @@ The comparison operators and logical operators are treated as expressions in
 Spark. Other than these two kinds of expressions, Spark supports other form of
 expressions such as function expressions, cast expressions, etc. The expressions
 in Spark can be broadly classified as :
-- Null in-tolerent expressions
+- Null in-tolerant expressions
```

Comment:
intolerant

Reply:
Fixed. Also fixed other usages in the same file.